src/types/Todo.ts

```diff
@@ -0,0 +1,7 @@
+export default interface Todo {
+  id: number
+  title: string
+  description: string
+  priority: 'Low' | 'Medium' | 'High'
+  done: boolean
```

Comment:
`isChecked` is a better variable name I think 😛 . Also please use `is` prefix for boolean variables.